### DISCUSSION TITLE
More Customization

### DISCUSSION THREAD

**Discussion by xGlitchette** [RoinujNosde/SimpleClans#206] [Ideas]
While the plugin is great, more customization option would, in my opinion, make the plugin much better for a server; I frankly don't like the way the GUI and Text look, and having them look nice is important to me. This is really my only issue with the plugin.

**Comment by RoinujNosde** [comment]
Every message is customizable. (https://github.com/RoinujNosde/SimpleClans/wiki/Editing-the-plugin's-messages)

What would you like to be customizable regarding the GUI?

**Comment by FireController1847** [comment]
Every message is customizable, but the formatting of many of the screens is not.

For example, if I want the invite screen to be exactly as so:
"You have been invited to <clan>. Run /allow or /deny"

As in one message with one color, I can't do this. Even though I can modify the messages, it's still prefixed with the [<CLAN>], and the /allow and /deny is still on an extra line (since I can't disable the second line, say, by leaving it blank). Similarly, the clan name in my message turns white for some reason, when I might want it to be a different color.

The format of many of the messages is non-customizable. For example, I don't want to show a player's rank upon doing /clan roster, but I have to. Or, say I don't want to show the KDR because I'm not using it in /clan stats, I can't do that either.

Even though all of the messages are customizable, the formatting is all over the place and makes it virtually impossible to make editing the messages useful.

**Comment by RoinujNosde** [comment]
Messages requiring better customization:
1. ~~Allow/deny messages~~
2. Table like commands (roster, stats, leaderboard, etc)
Anything else?

> Even though I can modify the messages, it's still prefixed with the []

Can you send me a screenshot of this?

**Comment by FireController1847** [comment]
Ask and you shall receive

![image](https://user-images.githubusercontent.com/16494272/89701371-84823600-d8f3-11ea-9c7f-87bb956d6042.png)

I modified the text to be gray and italic, that's it.


And if it's not clear, thank you for all of your work on this project. I appreciate all of your responses and your willingness to even think about implementing these features :)

**Comment by SrBedrock** [comment]
# has already been solved!

It would be nice to remove the formatting (text on the next line even with space left) from the messages and let the client handle it.

![image](https://user-images.githubusercontent.com/51332006/90017199-86e0e900-dc81-11ea-8fc7-e6fb486e202d.png)

It also happens with private messages from the clan and allies.
![image](https://user-images.githubusercontent.com/51332006/90303090-6e3f2180-de81-11ea-82ea-273e93602684.png)

**Comment by RoinujNosde** [comment]
Clan announcements are prefixed with an asterisk.

**Comment by RoinujNosde** [comment]
"None" in commands alliances, rivalries, vitals, profile is not translatable

**Comment by Tomut0** [comment]
Some info on hover, perhaps? 🤔 
I mean, let's say someone types a message:
> Minat0_ >> Hello there

What if add some information about this clan player on hovering his name? 
1. Clan: KoL
2. Rank: None
3. Status: Member
4. ...

Probably, it would be useless, because this information already appears in the chat. 🤔 
But if it would be customizable with PlaceholderAPI support, perhaps, it wouldn't be.
